templates/ctl-scaffold/README.md

```diff
@@ -0,0 +1,3 @@
+# ctl-scaffold
+
+TODO
```

Comment:
Please create an issue

Reply:
Oh sorry, that's from the old repo. We can just remove this